tests/test_implementations/test_semsimian_implementation.py

```diff
@@ -134,6 +136,29 @@ def test_all_by_all_pairwise_similarity(self):
             sem_similarity_object.phenodigm_score, sql_similarity_object.phenodigm_score, places=2
         )
 
+    def test_similarity_with_custom_ic_map(self):
+        adapter = self.oi
+
+        adapter.custom_ic_map_path = TEST_IC_MAP.as_posix()
+
+        if not isinstance(adapter, SemanticSimilarityInterface):
+            raise AssertionError("SemanticSimilarityInterface not implemented")
+        entities = [VACUOLE, ENDOMEMBRANE_SYSTEM]
+
+        for s in entities:
+            for o in entities:
+                for preds in [self.predicates]:
+                    sim = adapter.pairwise_similarity(s, o, predicates=preds)
+                    if sim is not None:
+                        if s == VACUOLE and o == VACUOLE:
+                            self.assertGreaterEqual(sim.ancestor_information_content, 5.5)
```

Comment:
Why not test for near-equality?

Reply:
This could actually just be equality as it's reading the score directly from the provided IC map.